ch04.adoc

```diff
@@ -91,11 +87,11 @@ float lon(lon) ;
 
 ====
 
-Application writers should note that the Udunits package has limited recognition of the directionality implied by the "east" part of the unit specification.
+Application writers should note that the UDUNITS package has limited recognition of the directionality implied by the "east" part of the unit specification.
 It defines **`degrees_east`** to be pi/180 radians, and hence equivalent to **`degrees_north`**.
 We recommend the determination that a coordinate is a longitude type should be done via a string match between the given unit and one of the acceptable forms of **`degrees_east`**.
 
-Optionally, the longitude type may be indicated additionally by providing the **`standard_name`** attribute with the value **`longitude`**, and/or the **`axis`** attribute with the value **`X`**.
+It is strongly recommended that the longitude type is also indicated by providing the **`standard_name`** attribute with the value **`longitude`**, and/or the **`axis`** attribute with the value **`X`**.
```

Comment:
I think this change (and the similar change to the latitude section), is substantive and also requires an update to the conformance document. So I shall revert this line back to the original.